app/decorators/placement_decorator.rb

```diff
@@ -35,4 +35,12 @@ def additional_subject_names
     reload unless new_record?
     additional_subjects.map(&:name).sort.to_sentence
   end
+
+  def term_names
+    if terms.any?
+      terms.order_by_term.pluck(:name).join(", ")
+    else
+      I18n.t("placements.schools.placements.show.any_time")
+    end
+  end
```

Comment:
This will conflict with @Kizr's https://github.com/DFE-Digital/itt-mentor-services/pull/968/files. Good idea to chat with him.